packages/js/src/dynamic-blocks.js

```diff
@@ -1,35 +1,24 @@
+/* External dependencies */
```

Comment:
I personally don't bother with these comments anymore and let my IDE auto-order imports.

Reply:
We don't enforce it anymore, right, but just for consistency with the other block-related scripts